int64 call
---
float64[27] boardpos
